### ISSUE TITLE
Cosmos Chain Connection Request - Maestro Chain

### ISSUE THREAD

**Issue by arifintahu** [axelarnetwork/support#144]
Hi last week, I have filled out the form of cosmos chain connection request Axelar Testnet x Maestro Chain Testnet on https://docs.axelar.dev/resources/ibc-chain-onboarding/
Any update on this?

**Comment by Olanetsoft** [comment]
Hello @arifintahu we are currently looking into it. Apologies for the delay

**Comment by arifintahu** [comment]
Alright, thanks for the update @Olanetsoft

**Comment by Olanetsoft** [comment]
> Alright, thanks for the update @Olanetsoft

Thanks for your patience @arifintahu 

We recently updated the chain onboarding process, and of course, this comes with more flexibility with integration, as you might have seen with our recent announcement [here](https://x.com/axelar/status/1841834338121290157). Introducing the [Mobius Development Stack](https://docs.axelar.dev/dev/what-is-mobius-development-stack/). We have a well-documented process [here](https://docs.axelar.dev/dev/amplifier/introduction/) that can guide you through the process. If you have any questions about integrations or support, let me know. I will be happy to help.

**Comment by arifintahu** [comment]
alright @Olanetsoft but jake said in discord that there's no integrating amplifier on ibc chain, all we need to do is just setup ibc connection, is it right?
![image](https://github.com/user-attachments/assets/c3f1ba55-5dd0-44ca-bf73-a8e78c93806d)

**Comment by Olanetsoft** [comment]
> alright @Olanetsoft but jake said in discord that there's no integrating amplifier on ibc chain, all we need to do is just setup ibc connection, is it right? ![image](https://private-user-images.githubusercontent.com/20294658/377684273-c3f1ba55-5dd0-44ca-bf73-a8e78c93806d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkyMTYzNzEsIm5iZiI6MTcyOTIxNjA3MSwicGF0aCI6Ii8yMDI5NDY1OC8zNzc2ODQyNzMtYzNmMWJhNTUtNWRkMC00NGNhLWJmNzMtYThlNzhjOTM4MDZkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE4VDAxNDc1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJkMTQ1YWNjYzZiNTlhZWUyNTA0ODQwMDYwNjY2NzYzOGI0ZTFjYTMwOTcwZDZjNGY3NmIwYjlmYTc0YmZkOGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.RdivlyBlPxfm1xQ356yLZ8cRzXKdvvhg5ShGJL-sSPQ)

Thats correct. Do you already have an IBC connection set via AXelar?

**Comment by arifintahu** [comment]
> > alright @Olanetsoft but jake said in discord that there's no integrating amplifier on ibc chain, all we need to do is just setup ibc connection, is it right? ![image](https://private-user-images.githubusercontent.com/20294658/377684273-c3f1ba55-5dd0-44ca-bf73-a8e78c93806d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjkyMTYzNzEsIm5iZiI6MTcyOTIxNjA3MSwicGF0aCI6Ii8yMDI5NDY1OC8zNzc2ODQyNzMtYzNmMWJhNTUtNWRkMC00NGNhLWJmNzMtYThlNzhjOTM4MDZkLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMTglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDE4VDAxNDc1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTJkMTQ1YWNjYzZiNTlhZWUyNTA0ODQwMDYwNjY2NzYzOGI0ZTFjYTMwOTcwZDZjNGY3NmIwYjlmYTc0YmZkOGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.RdivlyBlPxfm1xQ356yLZ8cRzXKdvvhg5ShGJL-sSPQ)
> 
> Thats correct. Do you already have an IBC connection set via AXelar?

yes, i have submitted to this [form](https://docs.google.com/forms/d/18as2Dn7mrcxoic1K4UwnqnViU6QQx3cSuduHBVqYNCQ/edit#response=ACYDBNilAOeWdjJ8cnWODTkG-srrvfmGbwB3gMEw-dUo34OV26h8sA9uNk7_JHSE2U6GO9U)

**Comment by arifintahu** [comment]
Hi @Olanetsoft , is there an estimated time for the connection request?

**Comment by Olanetsoft** [comment]
There isn't an ETA yet, but I will confirm this, sorry.
